src/sundials/fmod/CMakeLists.txt

```diff
@@ -19,8 +19,6 @@ set(sundials_SOURCES
   fsundials_adaptcontroller_mod.f90
   fsundials_context_mod.c
   fsundials_context_mod.f90
-  fsundials_errors_mod.c
-  fsundials_errors_mod.f90
```

Comment:
Yes, thats right. I wanted to avoid forcing users to have to `use` yet another module. I actually think we should combine all of the core modules into one to mimic how you would link to the libraries or include headers.  I have a branch where it is 95% done.